lib/segment/src/vector_storage/query/reco_query.rs

```diff
@@ -177,13 +177,17 @@ mod test {
 
             let ordering_before = float_cmp(dummy_similarity(&a), dummy_similarity(&b));
 
+            // Too similar scores can get compressed to the same value by the sigmoid function.
+            // This would make the test useless, so we skip those cases.
+            prop_assume!(ordering_before != Ordering::Equal);
```

Comment:
I wasn't aware of this macro. Cool stuff.

Reply:
I also learned about it today :D